language-support/ts/daml-json-types/BUILD.bazel

```diff
@@ -23,6 +25,14 @@ ts_library(
     ],
 )
 
+eslint_test(
+    name = "daml-json-types-lint",
+    srcs = glob(
+        ["**/*.ts"],
+        exclude = ["**/*.test.ts"],
```

Comment:
The `tsconfig.json` excludes the test files. So, we exclude them here as well. Otherwise, `eslint` complains.